lib/rules/security/func-visibility.js

```diff
@@ -61,18 +61,28 @@ class FuncVisibilityChecker extends BaseChecker {
       config && config.getObjectPropertyBoolean(ruleId, 'ignoreConstructors', false)
   }
 
+  ContractDefinition(node) {
+    this.isContract = node.kind === 'contract'
```

Comment:
This will always be true, because the `ContractDefinition` callback is called when a contract node is visited.

```suggestion
    this.isContract = true
```

Reply:
done